aggregate_temporal.json

```diff
@@ -22,39 +22,36 @@
         },
         {
             "name": "intervals",
-            "description": "Left-closed temporal intervals, which are allowed to overlap. Each temporal interval in the array has exactly two elements:\n\n1. The first element is the start of the temporal interval. The specified instance in time is **included** in the interval.\n2. The second element is the end of the temporal interval. The specified instance in time is **excluded** from the interval.\n\nThe specified temporal strings follow [RFC 3339](https://www.rfc-editor.org/rfc/rfc3339.html). Although [RFC 3339 prohibits the hour to be '24'](https://www.rfc-editor.org/rfc/rfc3339.html#section-5.7), **this process allows the value '24' for the hour** of an end time in order to make it possible that left-closed time intervals can fully cover the day.",
+            "description": "Left-closed temporal intervals, which are allowed to overlap. Each temporal interval in the array has exactly two elements:\n\n1. The first element is the start of the temporal interval. The specified time instant is **included** in the interval.\n2. The second element is the end of the temporal interval. The specified time instant is **excluded** from the interval.\n\nThe second element must always be greater/later than the first element, except for times. Otherwise, a `TemporalExtentEmpty` exception is thrown.",
             "schema": {
                 "type": "array",
                 "subtype": "temporal-intervals",
                 "minItems": 1,
                 "items": {
                     "type": "array",
                     "subtype": "temporal-interval",
+                    "uniqueItems": true,
                     "minItems": 2,
                     "maxItems": 2,
                     "items": {
                         "anyOf": [
                             {
                                 "type": "string",
                                 "format": "date-time",
-                                "subtype": "date-time"
+                                "subtype": "date-time",
+                                "description": "Date and time with a time zone."
```

Comment:
time zone is optional, right? (and assumed to be UTC when omitted)?

Reply:
No, it is required.